static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -146,6 +146,33 @@ define([
         }
     }
 
+    function initVideoHeadline(el, videoInfo) {
+        var buttonElement = el.parentElement.querySelector('button.vjs-big-play-button');
+        var hideControlBar = el.parentElement.querySelector('.vjs-control-bar');
+        // hideControlBar.classList.remove('vjs-control-bar');
+        hideControlBar.classList.add('vjs-control-bar--hidden');
+        buttonElement.classList.remove('vjs-big-play-button');
+        buttonElement.classList.add('vjs-big-play-button__duration');
+        var buttonDuration = el.getAttribute('data-formatted-duration');
+        buttonElement.dataset.formattedDuration = buttonDuration;
```

Comment:
`dataset` won't work on [IE < 11](http://caniuse.com/#search=dataset). Maybe consider using bonzo or `setAttribute`? Or target modern browsers in the test?